src/components/NavBar/NavigationBar.cy.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import NavigationBar from "@/components/NavBar/NavigationBar";
+import { AppRouterContext, AppRouterInstance } from "next/dist/shared/lib/app-router-context";
+
+describe("<NavigationBar />", () => {
+    const router: AppRouterInstance = {
+        push: () => {},
+        back: () => {},
+        forward: () => {},
+        replace: () => {},
+        prefetch: () => {},
+        refresh: () => {},
+    };
+
+    const RouterWrappedNavBar = (
+        <AppRouterContext.Provider value={router}>
+            <NavigationBar />
+        </AppRouterContext.Provider>
+    );
+
+    it("Renders on desktop", () => {
+        cy.viewport(1280, 800);
+        cy.mount(RouterWrappedNavBar);
+    });
+
+    it("Renders on mobile", () => {
+        cy.viewport(375, 667);
```

Comment:
for pixel values, can these be the actual minimum and maximum widths we are supporting? eg Mobile=(360,760) Desktop=(1366, 768) (random screens I found with dimensions linking the widths required)